app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageViewModel.kt

```diff
@@ -120,11 +121,13 @@ class NewMessageViewModel(application: Application) : AndroidViewModel(applicati
         }
     }
 
-    private fun MutableRealm.createDraft(draftMode: DraftMode, previousMessageUid: String?): Draft {
+    private fun MutableRealm.createDraft(draftMode: DraftMode, previousMessageUid: String?, recipient: Recipient?): Draft {
         return Draft().apply {
             initLocalValues(priority = Priority.NORMAL, mimeType = ClipDescription.MIMETYPE_TEXT_HTML)
             initSignature(realm = this@createDraft)
-            if (draftMode != DraftMode.NEW_MAIL) {
+            if (draftMode == DraftMode.NEW_MAIL) {
+                recipient?.let { to = realmListOf(it) }
+            } else {
```

Comment:
Wouldn't it be easier to read if we used a `when` statement so we can read the keywords `REPLY` and `REPLY_ALL`, or is it just me ? Because at first I was wondering where the reply part was